src/backend/marsha/core/api/video.py

```diff
@@ -232,10 +234,11 @@ def get_serializer_context(self):
             admin_role_permission = permissions.IsTokenAdmin()
             instructor_role_permission = permissions.IsTokenInstructor()
 
-            context["is_admin"] = admin_role_permission.has_permission(
-                request=self.request, view=None
-            ) or instructor_role_permission.has_permission(
-                request=self.request, view=None
+            context["is_admin"] = bool(
```

Comment:
Why casting a bool is needed here ?

Reply:
Because when it's not true it returns an empty `set`